test/test_sparse.py

```diff
@@ -1453,6 +1453,21 @@ def test_shape(num_mats, dim_i, dim_j, dim_k, nnz):
         test_shape(10, 10, 100, 0, 20)
         test_shape(10, 10, 100, 0, 20)
 
+    @onlyCUDA
+    @unittest.skipIf(
+        IS_WINDOWS and TEST_CUDA,
```

Comment:
Do we not already have a more specific condition for this? Like CUSPARSE_SPMM_COMPLEX128_SUPPORTED